vertx-db2-client/README.adoc

```diff
@@ -0,0 +1,45 @@
+= The Reactive DB2 Client
+
+== How to use it
+
+See `src/main/java/examples/DB2ClientExamples.java`
+
+== Testing
+
+Out of the box, the test suite runs a Docker container using https://www.testcontainers.org/[TestContainers] by default.
+
+Since DB2 container takes about 2 minutes to start, it is good to reuse containers across multiple runs. To container reuse, create a file at `~/.testcontainers.properties` and add the following property:
+
+.~/.testcontainers.properties
+```
+testcontainers.reuse.enable=true
+```
+
+Just run the following Maven command:
+
+```
+mvn test
+```
+
+== Testing with an external database
+
+If you want to use an external database, you must set the following environment variables:
+
+- DB2_HOST: The hostname of the machine where DB2 is
```

Comment:
are those the commonly used env names ?

for PG we do use `PGHOST`, etc... because those are _officially_ used see https://www.postgresql.org/docs/9.6/static/libpq-envars.html

just asking a question here